fcrepo-auth-webac/src/main/java/org/fcrepo/auth/webac/WebACRolesProvider.java

```diff
@@ -270,9 +270,14 @@
             triple -> triple.matches(triple.getSubject(), RDF_TYPE_NODE, VCARD_GROUP_NODE));
         //return members only if there is an associated vcard:Group
         if (hasVcardGroup) {
-            return triples.stream()
+            final List<String> values = new ArrayList();
+
+            final List<org.apache.jena.graph.Node>  objects = triples.stream()
                           .filter(triple -> triple.predicateMatches(VCARD_MEMBER_NODE))
-                          .map(Triple::getObject).flatMap(WebACRolesProvider::nodeToStringStream);
+                          .map(Triple::getObject).collect(Collectors.toList());
+            objects.stream().flatMap(WebACRolesProvider::nodeToStringStream).forEach(values::add);
+            objects.stream().flatMap(WebACRolesProvider::additionalAgentValues).forEach(values::add);
```

Comment:
If i'm understanding correctly, the stream returned by getAgentMembers is now going to contain the original URI of the agent and the URI of the agent relative to the repository, is that correct? Are both versions needed? Or is just the relativized version needed? 
Sorry, I'm not that familiar with the webac module, so I'm trying to get a handle on if the duplication is necessary. I guess what I'm wondering is if what is needed here is a combination of `nodeToStringStream` and `additionalAgentValue` so that the URI would only be added once and to keep down the number of iterations.
Bit of a tangent, but are `GROUP_AGENT_BASE_URI_PROPERTY` or `USER_AGENT_BASE_URI_PROPERTY` expected to be changeable while the repo is running? Wondering if they should be retrieved once at start time and reused.

Reply:
I believe @dbernstein is attempting to support both ACLs scenarios:
1. An ACL has Strings for user-ids (i.e. [additionalAgentValues](https://github.com/fcrepo4/fcrepo4/blob/master/fcrepo-auth-webac/src/main/java/org/fcrepo/auth/webac/WebACRolesProvider.java#L438))
```
vcard:hasMember "someuser"
```
2. An ACL has URIs for user-ids (i.e. [nodeToStringStream](https://github.com/fcrepo4/fcrepo4/blob/master/fcrepo-auth-webac/src/main/java/org/fcrepo/auth/webac/WebACRolesProvider.java#L285))
```
vcard:hasMember <http://example.com/someuser>
```

In both of the cases above, I believe we want the end result to be `someuser`. That being the case, it seems like the logic could be simplified to not include both:
```
someuser
http://example.com/someuser
```
..in the `values` List.

Specifically, if the `object` is a URI, we want to strip the URI prefix off (ala additionalAgentValues), we do not want to keep the URI (ala nodeToStringStream).

Reply:
Seeing that we were using both for acl:agent I thought there might be some reason for that.  I can change that, no problem.

Reply:
Re changeability of the system properties,  I believe they do need to be changeable for the integration tests.